addon-test-support/@ember/test-helpers/-internal/debug-info-helpers.ts

```diff
@@ -0,0 +1,29 @@
+export interface DebugInfoHelper {
+  name: string;
+  log: () => void;
+}
+
+export const debugInfoHelpers = new Set<DebugInfoHelper>();
+
+/**
+ * Registers a custom debug info helper to augment the output for test isolation validation.
+ *
+ * @public
```

Comment:
😱 :rage4: `*`

Reply:
Sorry, what's wrong here? I'm confused :)

Reply:
OH YOU KNOW WHAT YOU DID

Reply:
Hehe